addons/mail/static/src/models/thread/thread.js

```diff
@@ -1701,6 +1806,18 @@ function factory(dependencies) {
             compute: '_computeNeedactionMessages',
             dependencies: ['messages'],
         }),
+        /**
+         * Id of the message after which the new message separator should be placed.
+         * When undefined, this means the new message separator has to be placed before
+         * all messages.
```

Comment:
Change undefined to 0 in this doc string, and add comment to say undefined means there is no separator